onnxruntime/core/optimizer/qdq_transformer/relu_quantizelinear.cc

```diff
@@ -18,7 +18,8 @@ bool ReluQuantFusion::SatisfyCondition(const Graph& graph, const Node& node, con
 
   // if Relu is followed by QuantizeLinear, it can be fused into QuantizeLinear potentially
   const auto& next_node = *node.OutputNodesBegin();
-  if (!graph_utils::IsSupportedOptypeVersionAndDomain(next_node, "QuantizeLinear", {10, 13, 19})) {
+  if (!graph_utils::IsSupportedOptypeVersionAndDomain(next_node, "QuantizeLinear", {10, 13, 19}) &&
+      !graph_utils::IsSupportedOptypeVersionAndDomain(next_node, "QuantizeLinear", {1}, kMSDomain)) {
```

Comment:
`MatchQNode` is only defined for non-minimal builds, but ReluQuantFusion isn't guarded by the associated `#if !defined(ORT_MINIMAL_BUILD) || defined(ORT_EXTENDED_MINIMAL_BUILD)`. Is this something we need to adjust?

Reply:
many graph transformer source files are not included in minimal builds. we only need those ifdefs for ones that are.

I think this does not happen to be one of the transformers enabled in a minimal build, so that helper should be available.

https://github.com/microsoft/onnxruntime/blob/3cdf42548fcccaee05ce5a5e0f9ee563105c8665/cmake/onnxruntime_optimizer.cmake#L6-L61